Build fails on Xcode 10.2(Swift 5.0)

When I run `brew upgrade license-plist` I get the following error.

```
$ brew upgrade license-plist
==> Upgrading 1 outdated package:
mono0926/license-plist/license-plist 2.0.0 -> 2.3.0
==> Upgrading mono0926/license-plist/license-plist
==> Downloading https://github.com/mono0926/LicensePlist/archive/2.3.0.tar.gz
Already downloaded: /Users/davidcorbin/Library/Caches/Homebrew/downloads/fdbb23e4cbb10e97c178e8caf0d2aff6be4d3134ecf89b892773398384ccce25--LicensePlist-2.3.0.tar.gz
==> make install PREFIX=/usr/local/Cellar/license-plist/2.3.0
Last 15 lines from /Users/davidcorbin/Library/Logs/Homebrew/license-plist/01.make:
  "_swift_weakInit", referenced from:
      _$s6APIKit7SessionC14cancelRequests4with11passingTestyxm_SbxctAA7RequestRzlF in Session.swift.o
  "_swift_weakLoadStrong", referenced from:
      _$s6APIKit7SessionC14cancelRequests4with11passingTestyxm_SbxctAA7RequestRzlFySayAA0B4Task_pGcfU_SbAaH_pXEfU_ in Session.swift.o
  "_swift_willThrow", referenced from:
      _$s10Foundation4DataV6APIKitE11inputStream8capacityACSo07NSInputE0C_SitKcfCTf4gnd_n in Data+InputStream.swift.o
      _$s6APIKit28FormURLEncodedBodyParametersVAA0dE0A2aDP11buildEntityAA07RequestdG0OyKFTW in FormURLEncodedBodyParameters.swift.o
      _$s6APIKit18JSONBodyParametersV11buildEntityAA011RequestBodyE0OyKF in JSONBodyParameters.swift.o
      _$s6APIKit18JSONBodyParametersVAA04BodyC0A2aDP11buildEntityAA07RequestdF0OyKFTW in JSONBodyParameters.swift.o
      _$s6APIKit31MultipartFormDataBodyParametersVAA0eF0A2aDP11buildEntityAA07RequesteH0OyKFTW in MultipartFormDataBodyParameters.swift.o
      _$s6APIKit31MultipartFormDataBodyParametersV4PartV5value4name8mimeType8fileName8encodingAEyp_S2SSgAKSS10FoundationE8EncodingVtKcfC in MultipartFormDataBodyParameters.swift.o
      _$s6APIKit31MultipartFormDataBodyParametersV4PartV7fileURL4name8mimeType0H4NameAE10Foundation0I0V_S2SSgAMtKcfC in MultipartFormDataBodyParameters.swift.o
      ...
ld: symbol(s) not found for architecture x86_64
make: *** [build] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/mono0926/homebrew-license-plist/issues

```

I tried installing through homebrew again and got the same error with 2.4.0. 

 ```
$ brew upgrade license-plist
==> Upgrading 1 outdated package:
mono0926/license-plist/license-plist 2.0.0 -> 2.4.0
==> Upgrading mono0926/license-plist/license-plist
==> Downloading https://github.com/mono0926/LicensePlist/archive/2.4.0.tar.gz
Already downloaded: /Users/davidcorbin/Library/Caches/Homebrew/downloads/f895de1e973f820618d1fcbfd0aaedd10065f3f7841c28a69c2d40520c07de47--LicensePlist-2.4.0.tar.gz
==> make install PREFIX=/usr/local/Cellar/license-plist/2.4.0
Last 15 lines from /Users/davidcorbin/Library/Logs/Homebrew/license-plist/01.make:
  "_swift_weakInit", referenced from:
      _$s6APIKit7SessionC14cancelRequests4with11passingTestyxm_SbxctAA7RequestRzlF in Session.swift.o
  "_swift_weakLoadStrong", referenced from:
      _$s6APIKit7SessionC14cancelRequests4with11passingTestyxm_SbxctAA7RequestRzlFySayAA0B4Task_pGcfU_SbAaH_pXEfU_ in Session.swift.o
  "_swift_willThrow", referenced from:
      _$s10Foundation4DataV6APIKitE11inputStream8capacityACSo07NSInputE0C_SitKcfCTf4gnd_n in Data+InputStream.swift.o
      _$s6APIKit28FormURLEncodedBodyParametersVAA0dE0A2aDP11buildEntityAA07RequestdG0OyKFTW in FormURLEncodedBodyParameters.swift.o
      _$s6APIKit18JSONBodyParametersV11buildEntityAA011RequestBodyE0OyKF in JSONBodyParameters.swift.o
      _$s6APIKit18JSONBodyParametersVAA04BodyC0A2aDP11buildEntityAA07RequestdF0OyKFTW in JSONBodyParameters.swift.o
      _$s6APIKit31MultipartFormDataBodyParametersVAA0eF0A2aDP11buildEntityAA07RequesteH0OyKFTW in MultipartFormDataBodyParameters.swift.o
      _$s6APIKit31MultipartFormDataBodyParametersV4PartV5value4name8mimeType8fileName8encodingAEyp_S2SSgAKSS10FoundationE8EncodingVtKcfC in MultipartFormDataBodyParameters.swift.o
      _$s6APIKit31MultipartFormDataBodyParametersV4PartV7fileURL4name8mimeType0H4NameAE10Foundation0I0V_S2SSgAMtKcfC in MultipartFormDataBodyParameters.swift.o
      ...
ld: symbol(s) not found for architecture x86_64
make: *** [build] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/mono0926/homebrew-license-plist/issues
```

However, I cloned the repository and ran `make install` and there were no problems. I will just use the version from source for now! Thanks!

--- a/Makefile
+++ b/Makefile
@@ -3,6 +3,10 @@ PREFIX?=/usr/local
 TEMPORARY_FOLDER=./tmp_portable_licenseplist
 
 build:
+	swift build --disable-sandbox -c release
+
+# Xcode 10.1(Swift 4.2) or earlier
+build_legacy:
 	swift build --disable-sandbox -c release -Xswiftc -static-stdlib
 
 test:
@@ -21,7 +25,7 @@ install: build
 	mkdir -p "$(PREFIX)/bin"
 	cp -f ".build/release/license-plist" "$(PREFIX)/bin/license-plist"
 
-portable_zip: build
+portable_zip: build_legacy
 	mkdir -p "$(TEMPORARY_FOLDER)"
 	cp -f ".build/release/license-plist" "$(TEMPORARY_FOLDER)/license-plist"
 	cp -f "LICENSE" "$(TEMPORARY_FOLDER)"

--- a/Package.resolved
+++ b/Package.resolved
@@ -24,8 +24,8 @@
         "repositoryURL": "https://github.com/IBM-Swift/HeliumLogger.git",
         "state": {
           "branch": null,
-          "revision": "5ad546a39376b911a6cd4b06e8cadf60aa9e9472",
-          "version": "1.8.0"
+          "revision": "779865e83149a59894b14950aa83f70b7e81bc27",
+          "version": "1.8.1"
         }
       },
       {
@@ -33,8 +33,8 @@
         "repositoryURL": "https://github.com/IBM-Swift/LoggerAPI.git",
         "state": {
           "branch": null,
-          "revision": "d0ccb4158b83f8692531892a9c894bb757593c6f",
-          "version": "1.8.0"
+          "revision": "e29073bb7cecf3673e56bcb16180e8fd0cb091f6",
+          "version": "1.8.1"
         }
       },
       {
@@ -60,8 +60,8 @@
         "repositoryURL": "https://github.com/behrang/YamlSwift.git",
         "state": {
           "branch": null,
-          "revision": "978901aed4025e36ebf60c040c9000c559d7c3e0",
-          "version": "3.4.3"
+          "revision": "287f5cab7da0d92eb947b5fd8151b203ae04a9a3",
+          "version": "3.4.4"
         }
       }
     ]

--- a/Package.swift
+++ b/Package.swift
@@ -10,13 +10,13 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/kylef/Commander.git",
-                 from: "0.0.0"),
+                 from: "0.8.0"),
         .package(url: "https://github.com/ishkawa/APIKit.git",
                  from: "4.0.0"),
         .package(url: "https://github.com/IBM-Swift/HeliumLogger.git",
-                 from: "1.0.0"),
+                 from: "1.8.0"),
         .package(url: "https://github.com/behrang/YamlSwift.git",
-                 from: "3.0.0")
+                 from: "3.4.0")
     ],
     targets: [
         .target(

--- a/README.md
+++ b/README.md
@@ -7,6 +7,7 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-lightgrey.svg)](https://raw.githubusercontent.com/mono0926/NativePopup/master/LICENSE)
 [![Language: Swift 4.1](https://img.shields.io/badge/swift-4.1-4BC51D.svg?style=flat)](https://developer.apple.com/swift)
 [![Language: Swift 4.2](https://img.shields.io/badge/swift-4.2-4BC51D.svg?style=flat)](https://developer.apple.com/swift)
+[![Language: Swift 5.0](https://img.shields.io/badge/swift-5.0-4BC51D.svg?style=flat)](https://developer.apple.com/swift)
 [![Swift Package Manager compatible](https://img.shields.io/badge/Swift%20Package%20Manager-compatible-brightgreen.svg)](https://github.com/apple/swift-package-manager)
 
 `LicensePlist` is a command-line tool that automatically generates a Plist of all your dependencies, including files added manually(specified by [YAML config file](https://github.com/mono0926/LicensePlist/blob/master/Tests/LicensePlistTests/Resources/license_plist.yml)) or using `Carthage` or `CocoaPods`. All these licenses then show up in the Settings app.

--- a/release.sh
+++ b/release.sh
@@ -66,7 +66,8 @@ github-release upload \
 
 rm $lib_name.zip
 
-make portable_zip
+# CocoaPods
+DEVELOPER_DIR=/Applications/Xcode-10.1.app/Contents/Developer make portable_zip
 portable_zip_name="portable_licenseplist.zip"
 github-release upload \
     --user mono0926 \

--- a/release.sh
+++ b/release.sh
@@ -1,7 +1,14 @@
+#!/bin/sh
+
+# Check arguments
 if [ $# -eq 1 ]; then
     echo "A tag and token argument is needed!(ex: ./release.sh 1.2.3 xxxxxxx)"
     exit 1
 fi
+
+# build
+make build
+
 lib_name="license-plist"
 tag=$1
 token=$2
@@ -11,7 +18,7 @@ echo "Token: '${token}'"
 filename="${tag}.tar.gz"
 echo "Filename: '${filename}'"
 
-# tag
+# Push tag
 git tag $tag
 git push origin $tag
 
@@ -19,6 +26,7 @@ curl -LOk "https://github.com/mono0926/LicensePlist/archive/${filename}"
 sha256=$(shasum -a 256 $filename | cut -d ' ' -f 1)
 rm $filename
 
+# Homebrew
 formula_path="$lib_name.rb"
 formula_url="https://api.github.com/repos/mono0926/homebrew-$lib_name/contents/$formula_path"
 sha=`curl GET $formula_url \
@@ -43,6 +51,7 @@ curl -i -X PUT $formula_url \
 brew upgrade $lib_name
 zip -j $lib_name.zip /usr/local/bin/$lib_name
 
+# GitHub Release
 github-release release \
     --user mono0926 \
     --repo LicensePlist \

